samples/Experimental/VulkanRender/CMakeLists.txt

```diff
@@ -3,29 +3,12 @@ set(SOURCES
         )
 
 add_executable(VulkanRender ${SOURCES})
-add_dependencies(VulkanRender Dotnet Resources)
+add_dependencies(VulkanRender Resources)
```

Comment:
Uhhhhhhhhh.........why was this modified in this PR? Am I missing context?

Reply:
It was a missed change in another PR that was blocking me lol. I could have separated it but I didn't

Reply:
How is VulkanRender blocking you?

Reply:
Should be reverted as well